x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/util.ts

```diff
@@ -527,8 +553,167 @@ Example: Find the minimum between two fields averages
+    help: i18n.translate('xpack.lens.formula.eqFunction.markdown', {
+      defaultMessage: `
+Performs an equality comparison between two values.
+To be used as condition for \`ifelse\` comparison function.
+Also works with \`==\` symbol.
+
+Example: Returns true if the average of bytes is exactly the same amount of average memory
+\`average(bytes) == average(memory)\`
+
+Example: \`eq(sum(bytes), 1000000)\`
+    `,
+    }),
+  },
+  lte: {
+    section: 'comparison',
+    positionalArguments: [
+      {
+        name: i18n.translate('xpack.lens.formula.left', { defaultMessage: 'left' }),
+        type: getTypeI18n('number'),
+      },
+      {
+        name: i18n.translate('xpack.lens.formula.right', { defaultMessage: 'right' }),
+        type: getTypeI18n('number'),
+      },
+    ],
+    outputType: getTypeI18n('boolean'),
+    help: i18n.translate('xpack.lens.formula.lteFunction.markdown', {
+      defaultMessage: `
+Performs a lower than or equal comparison between two values.
+To be used as condition for \`ifelse\` comparison function.
+Also works with \`<=\` symbol.
+
+Example: Returns true if the average of bytes is lower than or equal to the average amount of memory
+\`average(bytes) <= average(memory)\`
+
+Example: \`lte(average(bytes), 1000)\`
+    `,
+    }),
+  },
+  gte: {
+    section: 'comparison',
+    positionalArguments: [
+      {
+        name: i18n.translate('xpack.lens.formula.left', { defaultMessage: 'left' }),
+        type: getTypeI18n('number'),
+      },
+      {
+        name: i18n.translate('xpack.lens.formula.right', { defaultMessage: 'right' }),
+        type: getTypeI18n('number'),
+      },
+    ],
+    outputType: getTypeI18n('boolean'),
+    help: i18n.translate('xpack.lens.formula.gteFunction.markdown', {
+      defaultMessage: `
+Performs a greater than comparison between two values.
+To be used as condition for \`ifelse\` comparison function.
+Also works with \`>=\` symbol.
+
+Example: Returns true if the average of bytes is greater than or equal to the average amount of memory
+\`average(bytes) >= average(memory)\`
+
+Example: \`gte(average(bytes), 1000)\`
+    `,
+    }),
+  },
+  ifelse: {
+    section: 'comparison',
+    positionalArguments: [
+      {
+        name: i18n.translate('xpack.lens.formula.condition', { defaultMessage: 'condition' }),
+        type: getTypeI18n('boolean'),
+      },
+      {
+        name: i18n.translate('xpack.lens.formula.left', { defaultMessage: 'left' }),
+        type: getTypeI18n('number'),
+      },
+      {
+        name: i18n.translate('xpack.lens.formula.right', { defaultMessage: 'right' }),
+        type: getTypeI18n('number'),
+      },
+    ],
+    help: i18n.translate('xpack.lens.formula.ifElseFunction.markdown', {
+      defaultMessage: `
+Returns a value depending on whether the element of condition is true or false.
+
+Example: Compare two fields average and return the highest
+\`ifelse( average(memory) >= average(bytes), average(memory), average(bytes))\`
```

Comment:
Not sure about this example, you can already do it via `pick_max`. What about the first example from the "Use cases" section here: https://github.com/elastic/kibana/issues/94603 ? Seems understandable to me.